.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 🐞 Bug Report
+about: Something is broken? 🔨
+---
+
+### Bug Report
+
+#### Summary
+
+<!-- Provide a summary describing the problem you are experiencing. -->
+
+#### Current behavior
+
+<!-- What is the current (buggy) behavior? -->
+
+#### How to reproduce
+
+<!--
+Provide steps to reproduce the bug.
+Adding a failing Unit or Functional Test would help us a lot - you can submit one in a Pull Request separately, referencing this bug report.
+-->
+
+#### Expected behavior
+
+<!-- What was the expected (correct) behavior? -->
+
+#### Please create a logfile using the following command and post the output here
+
+```
+$ embark run --nodashboard --logfile log.txt --loglevel trace
+```
```

Comment:
Nice, much better template, however we often end up asking information about the system that is why we used to have:
#### Context (Environment)
 * OS:
 * Embark Version:
 * Node Version:
 * NPM Version:

Do you think we should keep that part?

Reply:
We also don't ask for screenshot anymore and sometimes an image is better than a 1000s words :)

Reply:
Done! Please have another look.